tests/routes/validate_offerer_test.py

```diff
@@ -58,6 +61,65 @@ def expect_link_venue_to_iris_if_valid_to_have_been_called_for_every_venue(self,
             assert response.status_code == 202
             assert mocked_link_venue_to_iris_if_valid.call_count == 3
 
+        @patch('routes.validate.feature_queries.is_active', return_value=True)
+        @patch('routes.validate.redis.add_venue_id')
+        @clean_database
+        def expect_offerer_managed_venues_to_be_added_to_redis_when_feature_is_active(self, mocked_redis,
+                                                                                      mocked_feature, app):
+            # Given
+            offerer_token = secrets.token_urlsafe(20)
+            offerer = create_offerer(validation_token=offerer_token)
+            venue_1 = create_venue(offerer, idx=1)
+            venue_2 = create_venue(offerer, idx=2, siret=offerer.siren + '65371')
+            venue_3 = create_venue(offerer, idx=3, is_virtual=True, siret=None)
+            user = create_user()
+            user_offerer = create_user_offerer(user, offerer, is_admin=True)
+            repository.save(venue_1, venue_2, venue_3, user_offerer)
```

Comment:
```suggestion
            admin = create_user_offerer(user, offerer, is_admin=True)
            repository.save(admin)
```
Pas sûr de mon coup pour les venues...